cli/lib/snapshots.ts

```diff
@@ -28,14 +21,11 @@ export async function addPreviewSiteToAppdata(
 	previewUrl: string,
 	atomicSiteId: number,
 	siteFolder: string
-): Promise< void > {
+): Promise< Snapshot > {
 	try {
 		const userData = await readAppdata();
-		const site = userData.sites?.find( ( s ) => s.path === siteFolder );
-		if ( ! site ) {
-			return;
-		}
-		const snapshot: z.infer< typeof snapshotSchema > = {
+		const site = await getSiteFromFolder( siteFolder );
```

Comment:
This change means that this function now throws if the site isn't found in the appdata file.